toolchains/solidity/core/crates/linter-lib/src/ignore.rs

```diff
@@ -0,0 +1,35 @@
+use crate::errors::SolidHunterError;
+use glob::glob;
+use std::path::Path;
+
+fn parse_line(line: &str, path: &Path) -> Vec<String> {
+    let mut files = Vec::new();
+    let line = line.replace("./", "");
+    if let Some(parent) = path.parent() {
+        if let Some(filepath) = parent.join(line).to_str() {
+            if let Ok(entries) = glob(filepath) {
+                for entry in entries.flatten() {
+                    files.push(entry.into_os_string().into_string().unwrap())
+                }
+            }
+        }
+    }
+
+    files
+}
+
+pub fn get_ignored_files(filepath: &str) -> Result<Vec<String>, SolidHunterError> {
```

Comment:
Because we can have an error here https://github.com/astrodevs-labs/osmium/blob/121550971866ba4462aeeac943eb6488dc9ea581/toolchains/solidity/core/crates/linter-lib/src/ignore.rs#L29